ui/StatusQ/sandbox/sandboxapp.h

```diff
@@ -19,7 +19,7 @@ public slots:
     void restartEngine();
 
 private:
-    QQmlApplicationEngine m_engine;
+    QQmlApplicationEngine* m_engine {};
```

Comment:
`std::unique_ptr` for even more coolness :)

Reply:
oh, btw, I totally forgot to delete the engine on application close 🤦

Reply:
Which absolutely doesn't matter cos application closed? :D